internal/attestation/crafter/crafter.go

```diff
@@ -372,49 +372,43 @@ func persistCraftingState(craftState *api.CraftingState, stateFilePath string) e
 
 // ResolveEnvVars will iterate on the env vars in the allow list and resolve them from the system context
 // strict indicates if it should fail if any env variable can not be found
-func (c *Crafter) ResolveEnvVars(strict bool) error {
+func (c *Crafter) ResolveEnvVars() error {
 	if err := c.requireStateLoaded(); err != nil {
 		return err
 	}
 
-	// Runner specific env variables
+	// Runner specific environment variables
+
```

Comment:
I will change the comments to match what you like.

Just to feed your curiosity about my comment style, when the blocks below has empty lines in it, e.g.,
```
block_1_command_1()
block_1_command_2()

block_1_command3()
```
I do add an empty line around the comment, because otherwise it feels I am commenting only the first 2 commands :D So, instead of this:
```
// comment about block 1
block_1_command_1()
block_1_command_2()

block_1_command3()
```
I usually do this:
```
// comment about block 1

block_1_command_1()
block_1_command_2()

block_1_command3()
```